pyvista/plotting/opts.py

```diff
@@ -83,3 +83,18 @@ class PickerType(AnnotatedIntEnum):
     SCENE = (7, 'Scene')
     VOLUME = (8, 'Volume')
     WORLD = (9, 'World')
+
+
+class InteractionStyle(AnnotatedIntEnum):
```

Comment:
TL;DR, this needs a new name:  If in the future, we want to make the interaction styles _themselves_ to have a base class with subclasses, I think `InteractionStyle` (mirroring `vtkInteractionStyle`) would be a better name for that usage instead of the enum here.